### PR TITLE
PROVSUP-117 Fix incorrect variable used to set params sent to getSet().

### PR DESCRIPTION
**Pull request by bri-gaia**

### CODE REVIEW

File: app/controllers/manage/SetController.php [hunk 1]
```diff
@@ -109,17 +109,17 @@ public function ListSets() {
 			switch($pn_mode) {
 				case 0:
 				default:
-					$va_set_list['access'] = __CA_SET_EDIT_ACCESS__;
+					$set_params['access'] = __CA_SET_EDIT_ACCESS__;
```
Comment (bri-gaia) [inline app/controllers/manage/SetController.php:112]: @kehh lol, yeah, I can see how that would be an issue :sweat_smile:

**Comment by bri-gaia** [comment]
@kehh Re implemented sorts as per comments on !119 
https://github.com/gaiaresources/providence/pull/119#issuecomment-2378439616
https://github.com/gaiaresources/providence/pull/119#issuecomment-2378477844 

This time I thought I'd comment on the right PR! :P

File: app/controllers/manage/SetController.php [hunk 2]
```diff
@@ -109,21 +109,33 @@ public function ListSets() {
...
-					$va_set_list['publicUsers'] = true;
+					$set_params['publicUsers'] = true;
 					break;
 			}
 		} else {
-			$va_set_list['access'] = __CA_SET_EDIT_ACCESS__;
+			$set_params['access'] = __CA_SET_EDIT_ACCESS__;
 		}
 
+		$sql_sorts = array("cs.set_id", "csl.name", "clil.name_singular", "cs.access", "u.lname", "cs.access", "cs.status", "cs.rank");
```
Comment (kehh) [inline app/controllers/manage/SetController.php]: Remove duplicate element.
```suggestion
		$sql_sorts = array("cs.set_id", "csl.name", "clil.name_singular", "cs.access", "u.lname", "cs.status", "cs.rank");
```

Comment (kehh) [reply app/controllers/manage/SetController.php]: Well :octocat: if you want to treat the text change like that I guess you can :confounded:

**Pull request merged**